frontend/src/pages/Block.tsx

```diff
@@ -58,8 +60,17 @@ export const BlockPage: React.FC = () => {
 
   const isLoading = !block;
 
+  const title = useAppSelector(appTitle);
+  const favicon = useAppSelector(appFavicon);
+
+  const blockInfo = `${t('block-details')} | ${title}`;
+
   return (
     <PageWrapper error={error} isLoading={isLoading}>
+      <Helmet>
+        <link rel="icon" href={favicon} />
```

Comment:
do we need to redeclare this on every page? then maybe it's worth having a component `<HeaderMetadata title={"Blocks"}>` that will combine favicon data and `appTitle` in a form `${title} | ${appTime}`? WDYT?

Reply:
Created PageHeader component.